earlytracks-pythonjs-developper-job-advert-title-required.html

```diff
@@ -0,0 +1,59 @@
+---
+title: Python/JS Developper# Job Advert Title (required)
```

Comment:
You can get rid of the # and every character to the right on each line - they are just comments.